ouroboros-network/demo/chain-sync.hs

```diff
@@ -388,6 +388,10 @@ clientBlockFetch sockAddrs = do
         muxApplication peerid =
           MuxInitiatorApplication (protocols peerid)
 
+        prunePeerSTM :: FilePath -> STM ()
```

Comment:
`FilePath`?

Reply:
Surprising isn't it, it was forced by the caller, but we don't need to worry about this, this patch was deleted.